packages/account/src/Configs/personal-details-config.ts

```diff
@@ -210,9 +209,7 @@ export const personal_details_config = ({
         },
     };
 
-    // Need to check if client is high risk (only have SVG i.e. China & Russia)
-    // No need to get additinal details when client is high risk
-    if (!is_high_risk_client_for_mt5 && real_account_signup_target !== 'maltainvest') {
+    if (real_account_signup_target !== 'maltainvest') {
```

Comment:
Please update the test cases as well
<img width="829" alt="Screenshot 2024-03-20 at 10 19 35 AM" src="https://github.com/binary-com/deriv-app/assets/84661147/434f4dac-0886-4949-bbee-aa8554d3e6f0">